iothub/service/src/DigitalTwin/DigitalTwinClient.cs

```diff
@@ -105,6 +105,10 @@ public static DigitalTwinClient CreateFromConnectionString(string connectionStri
         /// <returns>An instance of <see cref="DigitalTwinClient"/>.</returns>
         /// <remarks>
         /// For more information on configuring IoT hub with Azure Active Directory, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-dev-guide-azure-ad-rbac"/>
+        /// This constructor internally uses <see cref="IotHubAuthenticationScopes.DefaultAuthenticationScopes"/>, which is used for
+        /// any public or private cloud other than Fairfax.
+        /// For Fairfax cloud users, use the constructor with <see cref="DigitalTwinClientOptions"/> and set the
```

Comment:
I had missed yesterday's conversation on the official name for Fairfax. I will be updating the full PR now by replacing it with AzureGoverment instead.